doc/news/changes.h

```diff
@@ -205,6 +205,20 @@ inconvenience this causes.
   (Luca Heltai, 2015/12/13)
   </li>
 
+  <li> New: A new linear operator representing the Schur complement,
+  namely schur_complement(), has been implemented. Some auxiliary functions
+  that are often used in conjunction with the Schur complement
+  (condense_schur_rhs() and postprocess_schur_solution()) are also provided
+  as a PackagedOperation.
+  An example of this functionality can be found in
+  <code>tests/lac/schur_complement_01.cc</code>.
+  The solution of a multiphysics problem (namely step-22) using the
```

Comment:
I wouldn't call step-22 a multiphysics problem (since it is 'only' Stokes), but rather a multi-component system.